src/pages/ProjectDetailPage/components/ProjectDetailContent/ProjectDetailTechStacks/ProjectDetailTechStacks.tsx

```diff
@@ -0,0 +1,53 @@
+// TODO: 1. 기술스택 이미지 URL 추가
+import { Flex, HStack, Text } from "@chakra-ui/react"
+import { TechStack } from "api-models"
+
+import CustomTag from "@components/Tag/components/CustomTag"
+
+import getGroupedCategory from "@pages/ProjectDetailPage/utils/getGroupedCategory"
+
+interface ProjectDetailTechStacksProps {
+  techStacks: TechStack[]
+}
+const ProjectDetailTechStacks = ({
+  techStacks,
+}: ProjectDetailTechStacksProps) => {
+  const groupedByCategory = getGroupedCategory(techStacks)
+
+  return (
+    <Flex
+      direction="column"
+      gap="3rem">
+      <Text
+        fontSize="2xl"
+        fontFamily="SCDream_Bold">
+        기술 스택
+      </Text>
+      <Flex
+        gap="3rem"
+        pl="5rem"
+        direction="column">
+        {groupedByCategory.map((category) => (
+          <Flex
+            key={category[0]}
+            direction="column"
+            gap="1rem">
+            <Text fontSize="xl">{category[0]}</Text>
```

Comment:
```suggestion
  const groupedByCategory = getGroupedCategory(techStacks)
  const [isLargerThan600] = useMediaQuery("(min-width: 600px)")

  return (
    <Flex
      direction="column"
      gap="3rem">
      <Text
        fontSize="2xl"
        fontFamily="SCDream_Bold">
        기술 스택
      </Text>
      <Flex
        gap="3rem"
        pl="5rem"
        direction="column">
        {groupedByCategory.map((category) => (
          <Flex
            key={category[0]}
           direction={isLargerThan600 ? "column" : "row"}
            gap="1rem">
            <Text
              fontSize="xl"
              whiteSpace="nowrap">
              {category[0]}
```
[P3] : chakra의 `useMediaQuery`로 600px 이하일 때는 direction을 row로 만들고
Text에는 whiteSpace를 `nowrap`으로 줘서 줄바꿈 안되게 만들어봤는데
이렇게하면 괜찮지 않을까요?

Reply:
> [P3] : chakra의 `useMediaQuery`로 600px 이하일 때는 direction을 row로 만들고 Text에는 whiteSpace를 `nowrap`으로 줘서 줄바꿈 안되게 만들어봤는데 이렇게하면 괜찮지 않을까요?

오 좋은방법같아요! 사실 카테고리 이름에 width를 고정해야한다는게 와닿지 않았는데 막상 해놓고 보니까 괜찮은것 같아요!! 감사합니다~~